.claude/scripts/ai-review/prioritizer.py

```
@@ -0,0 +1,299 @@
+        },
+        "Transaction 경계 위반": {
+            "pattern": r"@Transactional.*?(RestTemplate|WebClient|HttpClient|FeignClient)",
+            "reason": "Zero-Tolerance: @Transactional 내 외부 API 호출 금지"
+        },
+        "Long FK 위반": {
+            "pattern": r"@(ManyToOne|OneToMany|OneToOne|ManyToMany)",
+            "reason": "Zero-Tolerance: JPA 관계 어노테이션 금지 (Long FK 전략)"
+        }
+    }
+
+    def __init__(self, merged_issues: List[Dict]):
```

Ruff (0.14.3)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)